corehq/apps/app_manager/static/app_manager/js/summary/models.js

```diff
@@ -114,6 +114,7 @@ hqDefine('app_manager/js/summary/models',[
                     'question': gettext("Question"),
                     'name': gettext("Name"),
                     'label': gettext("Label"),
+                    'translations': gettext("Label"),
```

Comment:
Why doesn't this replace the `label` line above?

Reply:
good question - `label` is still used in some places, though I don't have a great understanding of where or why.  It could probably be removed from some (this one almost for sure).  I'll see if I can figure it out.